great_expectations/data_context/data_context.py

```diff
@@ -3317,6 +3317,15 @@ def add_profiler(
             ge_cloud_id=ge_cloud_id,
         )
 
+    def save_profiler(
+        self,
+        profiler: RuleBasedProfiler,
+        ge_cloud_id: Optional[str] = None,
+    ):
+        return profiler.save_profiler(
+            profiler_store=self.profiler_store, ge_cloud_id=ge_cloud_id
+        )
+
```

Comment:
Good question. I think since we already capture running a profiler with anonymized info about the configuration then that should be sufficient unless we really need to dive into save workflows e.g. to identify user issues. I can't @ Tanner here so will connect with him in slack for his opinion.

Reply:
@Super-Tanner

Reply:
my general philosophy is that if the marginal cost of implementing collection on new data points is low and there is clear business need for it, then go ahead. however, that does not seem to be the case at the moment: i get the impression it's a lot more work than we'd like it to be. i can't really speak to the business need. do you need to be able to measure if/when/how many contexts are using this method? i've worked with product managers who like to know how quickly new features are being adopted, for example, so they want to implement collection on literally every new event.